cwms-data-api/src/main/java/cwms/cda/api/RatingController.java

```diff
@@ -437,13 +436,18 @@ private String getRatingSetString(Context ctx, RatingSet.DatabaseLoadMethod meth
 
     private RatingSet getRatingSet(Context ctx, RatingSet.DatabaseLoadMethod method,
                                    String officeId, String rating, Instant begin,
-                                   Instant end) throws IOException, RatingException {
-        RatingSet ratingSet;
+                                   Instant end, boolean latest) throws IOException, RatingException {
```

Comment:
Don't add a boolean to a method to change the implementation, just create a new method

Reply:
Created a new method for latest rating retrieval